### PR TITLE
Emphasize differences in copr related permissions

### PR DESCRIPTION
**Pull request by majamassarini**

### CODE REVIEW

File: docs/configuration/upstream/copr_build.md [hunk 1]
```diff
@@ -64,10 +77,11 @@ that can change the behaviour of your Copr builds:
 * [**srpm_build_deps**](https://packit.dev/docs/configuration#srpm_build_deps)
 
 ## Using a custom Copr project
-When using a custom Copr project (by specifying `project` and `owner`), the GitHub repo has to be listed in the
-**Packit allowed forge projects** field in the **Copr project settings** so that the Copr builds can be actually run.
-As an example the string *github.com/osbuild/osbuild* has to be inserted
-into https://copr.fedorainfracloud.org/coprs/g/osbuild/osbuild/edit/#packit_forge_projects_allowed.
+When using a custom Copr project (by specifying `project` and `owner`) you need to:
+  - allow Packit to access your custom Copr project,
+  - allow builds from your project forge.
```
Comment (lbarcziova) [inline docs/configuration/upstream/copr_build.md:82]:
```suggestion
  - allow builds from your forge project.
```

File: docs/configuration/upstream/copr_build.md [hunk 2]
```diff
@@ -64,10 +77,11 @@ that can change the behaviour of your Copr builds:
 * [**srpm_build_deps**](https://packit.dev/docs/configuration#srpm_build_deps)
 
 ## Using a custom Copr project
-When using a custom Copr project (by specifying `project` and `owner`), the GitHub repo has to be listed in the
-**Packit allowed forge projects** field in the **Copr project settings** so that the Copr builds can be actually run.
-As an example the string *github.com/osbuild/osbuild* has to be inserted
-into https://copr.fedorainfracloud.org/coprs/g/osbuild/osbuild/edit/#packit_forge_projects_allowed.
+When using a custom Copr project (by specifying `project` and `owner`) you need to:
+  - allow Packit to access your custom Copr project,
```
Comment (lbarcziova) [inline docs/configuration/upstream/copr_build.md:81]:
```suggestion
  - allow Packit to build in your custom Copr project,
```

Comment (lbarcziova) [reply docs/configuration/upstream/copr_build.md:81]: and I would use the same for the subheader

**Review by lbarcziova — COMMENTED** [review]
thanks Maja, nice improvement!

**Comment by penguinpee** [comment]
Yes! Thank you. The screenshot in there really helps in drawing attention. Looks good from a (often lost) user's perspective.

**Review by lbarcziova — APPROVED** [review]

**Pull request merged**